glotaran/utils/test/test_io.py

```diff
@@ -1,6 +1,7 @@
 """Tests for glotaran/utils/io.py"""
 from __future__ import annotations
 
+import sys
```

Comment:
is there perhaps a bigger role for [pathlib](https://docs.python.org/3/library/pathlib.html) in lieu of sys?

Reply:
Fixed in 4d6301c0a66e31de369eba0c044c162f2f81c60e

Reply:
Could you elaborate more on what you mean?

Reply:
In this case sys is only used for host os resolving, so it's fine.